src/Plug.vala

```diff
@@ -92,7 +92,7 @@ namespace Network {
 
             no_devices = new Granite.Widgets.AlertView (
                 _("There is nothing to do"),
-                _("There are no available WiFi connections and devices connected to this computer.\n") +
+                _("There are no available Wi-Fi connections or -devices connected to this computer.\n") +
```

Comment:
accidentally added a "-" before devices?

Reply:
I'm not sure I really follow. "-devices" definitely looks like a typo. I'm not sure how adding a "-" is intended to change the meaning of the word

Reply:
@danrabbit Not quite sure if the convention works in English, I could make it "Wi-Fi connections or Wi-Fi devices"?